test/ansible-inventory/molecule/test-local/playbook.yml

```diff
@@ -16,7 +16,7 @@
     register: build_cmd
     changed_when: not prev_hash.stdout or (prev_hash.stdout and prev_hash.stdout not in ''.join(build_cmd.stdout_lines[-2:]))
 
-- name: Converge
+- name: Converge for Molecule Test
```

Comment:
And here

Reply:
However, when we are trying to follow the logs is hard to know from where the log came and has more than one ansible test with these names.

Reply:
I am closing it as solved since after our chat it shows make sense for you too.